binder/src/main/java/io/grpc/binder/SecurityPolicies.java

```diff
@@ -32,7 +32,6 @@
 import com.google.common.collect.ImmutableSet;
```

Comment:
Let's be more cautious with this class since I don't think we want to declare that none of these security policies will change. Which security policies do you specifically need to use from your library?

Reply:
We specifically need the `internalOnly()` public api. The annotation has been applied to the other public APIs.